extensions/wikia/EditPageLayout/js/editor/WikiaEditor.js

```diff
@@ -1069,14 +1074,15 @@
 		stateProxiedCommands: {},
 
 		beforeInit: function() {
+			
 			this.editor.ui.addExternalProvider(this);
 			this.editor.on('ck-themeLoaded',this.ckReady,this);
 			this.editor.on('uiBuildClickHandler',this.buildWysiwygClickHandler,this);
 			this.editor.on('uiStandardElementCreated',this.elementCreated,this);
 			this.editor.on('mode',this.modeChanged,this);
 		},
 
-		modeChanged: function( editor, mode ) {
+		modeChanged: function( editor, mode ) {	
 			// show/hide appropriate buttons
 			for (var name in this.modeAwareCommands) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Restrict what this loop acts on by testing each property. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AForIn)

[project-id]: wikia:app